checkov/common/bridgecrew/image_scanning/twistlock_integration.py

```diff
@@ -0,0 +1,34 @@
+import logging
+import os
+import platform
+import stat
+import requests
+
+from checkov.common.bridgecrew.platform_integration import bc_integration
+from checkov.common.bridgecrew.integration_features.base_integration_feature import BC_API_URL
+from checkov.common.util.dict_utils import merge_dicts
+from checkov.common.util.http_utils import get_auth_header, get_default_get_headers
+
+
+class TwistLockIntegration:
+    twistlock_base_url = f"{BC_API_URL}/vulnerabilities/twistlock"
+
+    def get_bc_api_key(self):
+        return bc_integration.bc_api_key
+
+    def get_proxy_address(self):
+        return f"{self.twistlock_base_url}/proxy"
+
+    def download_cli(self, twistlock_cli_file_name):
+        os_type = platform.system().lower()
```

Comment:
what happens in case of no access to internet?

Reply:
The scan will fail. What is the expected behavior?

Reply:
what's the printed output? can you try and share?

Reply:
It fails in a prior phase.
![image](https://user-images.githubusercontent.com/1766724/114526810-fdd6e300-9c4f-11eb-8e84-f1c1ba6c96a0.png)

Reply:
@elaygl Today checkov supports running without internet access... Need to find a way to handle it gracefully here as well

Reply:
@schosterbarak @nimrodkor Currently, when Checkov is provided with `--bc-api-key` it requires internet. This PR is not a regression. 
If you want to handle this case, we can create another PR for that.